src/config.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"github.com/HyperloopUPV-H8/Backend-H8/connection_transfer"
+	"github.com/HyperloopUPV-H8/Backend-H8/data_transfer"
+	"github.com/HyperloopUPV-H8/Backend-H8/excel_adapter"
+	"github.com/HyperloopUPV-H8/Backend-H8/log_handle"
+	"github.com/HyperloopUPV-H8/Backend-H8/message_transfer"
+	"github.com/HyperloopUPV-H8/Backend-H8/server"
+	"github.com/HyperloopUPV-H8/Backend-H8/vehicle"
+)
+
+type Config struct {
```

Comment:
have this as a singleton that is initialized from main, then provide getter methods to access module configuration.

This way you dont need to pass the config to all methods, it can be accessed globally.